index.html

```diff
@@ -3,10 +3,11 @@
 <head>
  
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
- <title>JS</title>
+<title>JS</title>
 <meta name="keywords" content="" />
 <link href="css/style.css" type="text/css" rel="stylesheet"  media="screen" />
- 
+<script type="text/javascript" src="js/js.js"></script>

```

Comment:
@burymm I shifted the script and remove unnecessary. while the rest don't know what to do.